src/docstringtemplate.jl

```diff
@@ -5,46 +5,48 @@ function _docstring(cfg_symb::Symbol)
     out = ""
 
     visuallink = Dict(
-        :erp =>`Makie.lines`,
-        :butterfly =>`Makie.lines`,
-        :paracoord =>`Makie.lines`,
-        :erpgrid =>`Makie.lines`,
+        :erp => `Makie.lines`,
+        :butterfly => `Makie.lines`,
+        :paracoord => `Makie.lines`,
+        :erpgrid => `Makie.lines`,
         :designmat => `Makie.heatmap`,
         :erpimage => `Makie.heatmap`,
         :channelimage => `Makie.heatmap`,
         :circeegtopo => `Topoplot.eeg_topoplot`,
         :topoplot => `Topoplot.eeg_topoplot`,
-        :topoplotseries => `Topoplot.eeg_topoplot`
+        :topoplotseries => `Topoplot.eeg_topoplot`,
     )
-    cbarstring = (cfg_symb == :erp || cfg_symb == :butterfly) ? "[`AlgebraOfGraphics.colobar!`](@ref)" : "[`Makie.Colorbar`](@ref)"
+    cbarstring =
+        (cfg_symb == :erp || cfg_symb == :butterfly) ?
+        "[`AlgebraOfGraphics.colobar!`](@ref)" : "[`Makie.Colorbar`](@ref)"
     link = Dict(
         :figure => "use `kwargs...` of [`Makie.Figure`](@ref)",
         :axis => "use `kwargs...` of  [`Makie.Axis`](@ref)",
         :legend => "use `kwargs...` of  [`Makie.Legend`](@ref)",
         :colorbar => "use `kwargs...` of  $cbarstring",
         :visual => "use `kwargs...` of [`$(visuallink[cfg_symb])`](@ref)",
-        
     )
     for k = 1:length(fn)
         namedtpl = Base.getfield(cfg, fn[k])
         addlink = ""
         try
-            addlink = "- *"*link[fn[k]]*"*"
+            addlink = "- *" * link[fn[k]] * "*"
```

Comment:
this supposedly adresses the italic, but I think the wrong italic. the issue is this italic:
![grafik](https://github.com/unfoldtoolbox/UnfoldMakie.jl/assets/10183650/5cc33a4b-163e-4e95-a258-377bf1eba7a0)

not this 
![grafik](https://github.com/unfoldtoolbox/UnfoldMakie.jl/assets/10183650/9e0c2780-2e4c-4069-b383-fb42e5522ea2)


I'd rather keep the latter actually.

Reply:
![image](https://github.com/unfoldtoolbox/UnfoldMakie.jl/assets/33777074/09e69154-c1d0-4fb4-b610-91b7c459aac5)

the solution is on the line 37
on 32 is just an action of JuliaFormatter

Reply:
no. formater broke it that is why you dont see the italics ;)

Reply:
that is also why it now says axis = namedtpl instead of the actual content

Reply:
ah I see, indeed
any ideas how to solve it?

Reply:
definitely revert the formatter change - I have to think about it